app/views/layouts/application.html.erb

```diff
@@ -19,7 +19,7 @@
         <% end %>
         <span class="caret"></span>
       </a>
-      <ul class="dropdown-menu" role="menu">
+      <ul class="dropdown-menu finders" role="menu">
```

Comment:
You could always try to find the first `dropdown-menu` if someone adds another one before then your test will fail, but that might be as it will be meant to and possibly force the next person to add another test for it's own `dropdown-menu`

I can think of two possible options to do that `:first-child` or `.dropdown-menu:nth-of-type(1)`